src/Console/Command.php

```diff
@@ -5,36 +5,66 @@
 use Sokil\Mongo\Migrator\Config;
 use Sokil\Mongo\Migrator\Console\Exception\ConfigurationNotFound;
 use Sokil\Mongo\Migrator\Manager;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Yaml\Yaml;
 
 abstract class Command extends \Symfony\Component\Console\Command\Command
 {
+    /**
+     * @var Config
+     */
     private $config;
-    
+
     /**
      *
      * @var \Sokil\Mongo\Migrator\Manager
      */
     private $manager;
     
     const CONFIG_FILENAME = 'mongo-migrator';
-    
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @throws ConfigurationNotFound
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        // config file path
+        $configPath = $input->getOption('configuration');
+
+        if (!empty($configPath)) {
+            $this->config = new Config($this->readConfig($configPath));
```

Comment:
you need to init config there in any case, and `getConfig` must just return this value, without lazy reading

Reply:
you want to remove this if and leave `$this->config = ..` without it?